ifrs17-template/Test/ScenarioDataImportTest.ipynb

```diff
@@ -0,0 +1,868 @@
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var cashflowsScenarioEqualToBestEstimate = @\"",
+                "\n@@Main",
+                "\nReportingNode,Year,Month,Scenario",
+                "\nCH,2020,12,Test",
+                "\n@@Cashflow",
+                "\nDataNode,AmountType,AocType,Novelty,AccidentYear,Values0,Values1,Values2,Values3,Values4,Values5,Values6,Values7,Values8,Values9,Values10,Values11,Values12,Values13,Values14,Values15,Values16,Values17,Values18,Values19,Values20,Values21,Values22,Values23",
+                "\nDT1.1,PR,CL,C,,100,0,0,100,0,0,100,0,0,100,0,0,0,100,0,0,100,0,0,100,0,0,100,0",
+                "\nDT1.1,NIC,CL,C,,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25,-25",
+                "\nDT1.1,CU,CL,C,,-10,-10,-10,-10,-10,-10,-10,-10,-10,-10,-10,-10,-10,-5,-5,-5,-5,-5,-5,-5,-5,-5,-5,-3",
+                "\nDT1.1,RA,CL,C,,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5,-2.5\";"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var actualsScenarioEqualToBestEstimate = @\"",
+                "\n@@Main",
+                "\nReportingNode,Year,Month,Scenario",
+                "\nCH,2020,12,Test",
+                "\n@@Actual",
+                "\nDataNode,AocType,ValueType,AccidentYear,Value",
+                "\nDT1.1,CF,ACA,,-10",
+                "\nDT1.1,CF,AEA,,-5",
+                "\nDT1.1,CF,NIC,,-280\";"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await Import.FromString(actualsScenarioEqualToBestEstimate).WithFormat(ImportFormats.Actual).WithTarget(DataSource).ExecuteAsync()"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await Import.FromString(cashflowsScenarioEqualToBestEstimate).WithFormat(ImportFormats.Cashflow).WithTarget(DataSource).ExecuteAsync()"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "await DataSource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(argsScenario);",
+                "\nvar ivsScenarioReimportEqualToBestEstimate = await DataSource.Query<IfrsVariable>().ToArrayAsync(); ",
+                "\nvar diffs = ivsScenarioReimportEqualToBestEstimate.Except(ivsBestEstimateInit, IfrsVariableComparer.Instance()).Where(x => Math.Abs(x.Value) > Precision);"
```

Comment:
Also here, if we save the 0s correctly we should not have any difference: 
remove .Where(x => Math.Abs(x.Value) > Precision)

Reply:
here we are comparing Scenario IfrsVariables with BestEstimate ones. If we don't exclude zeros, the test fails since it cannot match any zeros.